### PR TITLE
{bio}[dummy] sratoolkit 2.5.7 

### PR DESCRIPTION
**Pull request by verdurin**
This is an attempt to address the maintainability concerns noted in #1790

### CODE REVIEW

**Comment by boegel** [comment]
Jenkins: ok to test

File: easybuild/easyconfigs/s/sratoolkit/sratoolkit-2.5.7.eb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+executables = [
+'abi-dump*', 'abi-load*', 'align-info*', 'bam-load*', 'blastn_vdb*', 'cache-mgr*', 
+'cg-load*', 'fastq-dump*', 'fastq-load*', 'helicos-load*', 'illumina-dump*', 
+'illumina-load*', 'kar*', 'kdbmeta*', 'latf-load*', 'ncbi', 'pacbio-load*', 
+'prefetch*', 'rcexplain*', 'remote-fuser*', 'sam-dump*', 'sff-dump*', 
+'sff-load*', 'sra-kar*', 'sra-pileup*', 'sra-sort*', 'sra-stat*', 'srapath*', 
+'srf-load*', 'tblastn_vdb*', 'test-sra*', 'vdb-config*', 'vdb-copy*', 
+'vdb-decrypt*', 'vdb-dump*', 'vdb-encrypt*', 'vdb-lock*', 'vdb-passwd*', 
+'vdb-unlock*', 'vdb-validate*',
```
Comment (boegel) [inline easybuild/easyconfigs/s/sratoolkit/sratoolkit-2.5.7.eb]: please fix indent (4 spaces)?

File: easybuild/easyconfigs/s/sratoolkit/sratoolkit-2.5.7.eb [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+]
+
+files_to_copy = [('*', 'bin')]
+
+sanity_check_paths = {
+    'files': ['bin/abi-dump', 'bin/illumina-load', 'bin/srf-load', ],
```
Comment (boegel) [inline easybuild/easyconfigs/s/sratoolkit/sratoolkit-2.5.7.eb]: or just check for a non-empty `bin` (by including it in the `dirs` part below)?

**Comment by verdurin** [comment]
@boegel - better?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2111.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/2634ce4508db0bb60e2f for a full test report.

File: easybuild/easyconfigs/s/sratoolkit/sratoolkit-2.5.7.eb [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+description = """The SRA Toolkit, and the source-code SRA System Development 
+ Kit (SDK), will allow you to programmatically access data housed within SRA
+ and convert it from the SRA format"""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['http://ftp-trace.ncbi.nlm.nih.gov/sra/sdk/%(version)s/']
+sources = ['sratoolkit.%(version)s-centos_linux64.tar.gz']
+
+executables = [
```
Comment (boegel) [inline easybuild/easyconfigs/s/sratoolkit/sratoolkit-2.5.7.eb]: wait, this list isn't used anywhere now?!

File: easybuild/easyconfigs/s/sratoolkit/sratoolkit-2.5.7.eb [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    'cg-load*', 'fastq-dump*', 'fastq-load*', 'helicos-load*', 'illumina-dump*', 
+    'illumina-load*', 'kar*', 'kdbmeta*', 'latf-load*', 'ncbi', 'pacbio-load*', 
+    'prefetch*', 'rcexplain*', 'remote-fuser*', 'sam-dump*', 'sff-dump*', 
+    'sff-load*', 'sra-kar*', 'sra-pileup*', 'sra-sort*', 'sra-stat*', 'srapath*', 
+    'srf-load*', 'tblastn_vdb*', 'test-sra*', 'vdb-config*', 'vdb-copy*', 
+    'vdb-decrypt*', 'vdb-dump*', 'vdb-encrypt*', 'vdb-lock*', 'vdb-passwd*', 
+    'vdb-unlock*', 'vdb-validate*',
+]
+
+files_to_copy = [('*', 'bin')]
```
Comment (boegel) [inline easybuild/easyconfigs/s/sratoolkit/sratoolkit-2.5.7.eb]: `PackedBinary` doesn't use `files_to_copy`... :)

File: easybuild/easyconfigs/s/sratoolkit/sratoolkit-2.5.7.eb (hunk 3 above)
Comment (verdurin) [inline easybuild/easyconfigs/s/sratoolkit/sratoolkit-2.5.7.eb]: Ah, I had assumed it was required and just re-used for convenience in the original PR.

Old RPM habits die hard, I suppose.

File: easybuild/easyconfigs/s/sratoolkit/sratoolkit-2.5.7.eb (hunk 4 above)
Comment (verdurin) [inline easybuild/easyconfigs/s/sratoolkit/sratoolkit-2.5.7.eb]: @boegel - shall I just remove those two stanzas then?

Is that what's expected with `PackedBinary`?

Comment (boegel) [inline easybuild/easyconfigs/s/sratoolkit/sratoolkit-2.5.7.eb]: yeah, there's no point in having those here, they're not used

that does imply we should make the `sanity_check_paths` below a bit more specific though, like checking for a handful of binaries (like you had before)

if we're copying them ourselves, there's little point in checking them again afterwards, but that's different when just relying on `PackedBinary` imho

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2111.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/b79e725602e453713419 for a full test report.

**Comment by boegel** [comment]
lgtm, thanks @verdurin!

**Pull request merged**